src/nft/types.ts

```diff
@@ -130,8 +135,8 @@ export type NFTMintCardReact = {
   contractAddress: Hex; // Contract address of the NFT
   tokenId?: string; // Token ID of the NFT only required for ERC1155
   isSponsored?: boolean; // Optional boolean to determine if the mint is sponsored by paymaster
-  useNFTData: UseNFTData; // Required hook to supply NFT data
-  buildMintTransaction: BuildMintTransaction; // Required function that builds the mint transaction
+  useNFTData?: UseNFTData; // Required hook to supply NFT data
+  buildMintTransaction?: BuildMintTransaction; // Required function that builds the mint transaction
```

Comment:
A little confused why these params are marked as optional the comments state that they are required

Reply:
good catch, updated the params, but not the comments, they used to be required.

Reply:
fixed